### PR TITLE
resolve #71 Event cover image upload

### LINKED ISSUE #71: Event cover image upload

### PR DESCRIPTION
**Pull request by denys-medynskyi**
Fixed #71
@denysmedynskyi just add text like this here

### CODE REVIEW

**Comment by sveredyuk** [comment]
good job

**Comment by sveredyuk** [comment]
@denysmedynskyi at least 1 little spec...

**Review by sveredyuk — APPROVED** [review]

File: app/uploaders/event_cover_uploader.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  # include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
+  # storage :fog
+
+  # Override the directory where uploaded files will be stored.
+  # This is a sensible default for uploaders that are meant to be mounted:
+  def store_dir
+    "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
```
Comment (fuelen) [inline app/uploaders/event_cover_uploader.rb]: add test subdirectory for test environment

**Comment by denys-medynskyi** [comment]
@sveredyuk there is feature spec, which checks if image is uploaded to event

File: app/uploaders/event_cover_uploader.rb (hunk 1 above)
Comment (denys-medynskyi) [reply app/uploaders/event_cover_uploader.rb]: @fuelen thanks. done.

**Comment by denys-medynskyi** [comment]
@sveredyuk code coverage is fixed

**Pull request merged**